lib/Widgets/DynamicNotebook.vala

```diff
@@ -281,42 +296,32 @@ namespace Granite.Widgets {
             duplicate_m.activate.connect (() => duplicate () );
             pin_m.activate.connect (() => pinned = !pinned);
 
+            add_events (Gdk.EventMask.SCROLL_MASK);
```

Comment:
Just for clarification -- does that mean we had code to handle scroll_events, but they never arrived because of the missing mask?

Reply:
as far as I understand it, yes! Would it maybe better to remove it in this PR and merge the other PR for cleaner commit history?

Reply:
I removed the scroll-able tabs from this PR as there is already a separate PR (#303) which will enable this behavior.

Reply:
Never mind, I think it is better to include the scroll-able tabs in this PR, because #303 would raise a merge conflict as does not has the new `dynamic_notebook` property of this PR.